corehq/motech/repeaters/management/commands/generate_repeaters_summary.py

```diff
@@ -16,18 +16,17 @@ def handle(self, *args, **options):
 
         self.stdout.write("\n")
         self.stdout.write('fetching repeat record data...')
-        repeat_records_summary = RepeatRecord.get_db().view(
-            'repeaters/repeat_records',
-            group_level=1,
-            reduce=True
-        ).all()
+        repeat_records_summary = dict(
+            SQLRepeatRecord.objects
+            .values("repeater__domain")
+            .annotate(record_count=Count("id"))
+            .values_list("repeater__domain", "record_count")
+        )
```

Comment:
@nospame and I paired on this, and only got as far as verifying this bug 😅. Does this sound right to you @millerdev?

The current query doesn't behave the same as the couch query it is replacing. Upon further inspection, this query was also grouping by "registered_at":
```
In [46]: print(SQLRepeatRecord.objects.values("repeater__domain").annotate(record_count=Count("id")).values_list("repeater__domain", "record_count").query)
SELECT "repeaters_repeater"."domain", COUNT("repeaters_repeatrecord"."id") AS "record_count" FROM "repeaters_repeatrecord" INNER JOIN "repeaters_repeater" ON ("repeaters_repeatrecord"."repeater_id_" = "repeaters_repeater"."id_") GROUP BY "repeaters_repeater"."domain", "repeaters_repeatrecord"."registered_at"
```

I think this is due to `registered_at` being the default ordering of results (specified [here](https://github.com/dimagi/commcare-hq/blob/108bf24558fe6477157a7d5b5d3accfacda486d7/corehq/motech/repeaters/models.py#L1376-L1376)), so to resolve, you need to `order_by` nothing to remove the `registered_at` ordering, like so:
```
In [47]: print(SQLRepeatRecord.objects.values("repeater__domain").order_by().annotate(record_count=Count("id")).values_list("repeater__domain", "record_count").query)
SELECT "repeaters_repeater"."domain", COUNT("repeaters_repeatrecord"."id") AS "record_count" FROM "repeaters_repeatrecord" INNER JOIN "repeaters_repeater" ON ("repeaters_repeatrecord"."repeater_id_" = "repeaters_repeater"."id_") GROUP BY "repeaters_repeater"."domain"
```

Reply:
Yeah we were just trying to understand the query better since neither of us were super confident with the annotation aspect, and noticed that the result we got back was surprising!

Reply:
ccb2ce2792156e155d6903eb214aa57babd4199e